index.html

```diff
@@ -0,0 +1,70 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;1,300&display=swap" rel="stylesheet">
+  <title>Mint Signup Clone</title>
+</head>
+
+<body>
+
+<header>
+  <div class='top-buttons'>
+    <ul>
+      <li><span>Already have an Intuit account?</span></li>
+      <li><a href="#forgot">I forgot my User ID or Password</a></li>
+    </ul>
+    <button>
+      <img src="assets/lock.svg" alt='lock icon'>
+      Sign In
+    </button>
+  </div>
+</header>
+
+<main>
+
+  <img src="assets/intuit-logo.png" id='logos' alt="intuit logo">
+  
+  <form action='/signup' method="POST" class="signup-form">
+    <h2>Create an Intuit account</h2>
+    <p>One account for everything Intuit, including Mint.</p>
+    <a>Learn more</a>
+    <label>
+      Email address
+      <input type="email" name="email">
+    </label>  
+    <label>
+      Phone (recommended)
+      <input type='tel' name="phone">
+      <sub>Standard call, messaging or data rates may apply.</sub>
+    </label>
+    <label>
+      Password
+      <input type='password' name="password">
+    </label>
+    <button><img src="assets/lock.svg" alt='lock icon'> Create Account</button>
```

Comment:
The button should have a type='submit':
- [ ] It has a `<submit>` button that matches the style of the original page (Blue background, white text) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))